src/components/AuthenticatedAppHeader/NotificationsPane.jsx

```diff
@@ -116,7 +117,11 @@ export default function NotificationsPane({
                       await markRead(get(notification, 'guid'));
                       refreshNotifications();
                     }}
-                    style={{ maxHeight: 40 }}
+                    style={{
+                      maxHeight: 40,
+                      display:
+                        currentNotificationSchema?.displayViewMoreInfoButton,
```

Comment:
While I think the setup for the display text is good, I'm not sure this is the right way to do the buttons. If a view button isn't part of the notification at all, it shouldn't be rendered to the DOM. (Here it is rendered with `display: none`.)

Instead it could be like this:`{displayViewMoreInfoButton && <Button id="VIEW" ... />}`

Reply:
Moot because of below.